ATSQuadroStrategyBase/NinjaTrader 8/bin/Custom/Strategies/AlgoSystemBase.cs

```diff
@@ -2567,7 +2567,7 @@ public virtual StrategyTradeWorkFlowState ProcessWorkFlow(StrategyTradeWorkFlowS
                     return ProcessWorkFlow(StrategyTradeWorkFlowState.GoShortSubmitOrder);
 
                 case StrategyTradeWorkFlowState.GoShortCancelWorkingOrders:
-                    if (IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode)

```

Comment:
This is required - IsHistoricalTradeOrPlayBack || IsStrategyUnSafeMode use the same paths a lot of the time-